### ISSUE TITLE
Palindromes tests fail online but pass locally

### ISSUE THREAD

**Issue by belgoros** [exercism/ruby-test-runner#58]
I hit an issue with failing tests for [Palidrome Products](https://exercism.org/tracks/ruby/exercises/palindrome-products) exercise: all the tests pass locally but fail online:

<img width="523" alt="Screenshot 2022-07-18 at 13 45 56" src="https://user-images.githubusercontent.com/83579/179504587-a5f4ad6e-b201-411e-aba1-7a776f80aebb.png">

As you could see it passes with Ruby `2.7.6`.

I also tried the latest Ruby version, `3.1.2`, - it also passed:
<img width="515" alt="Screenshot 2022-07-18 at 13 49 59" src="https://user-images.githubusercontent.com/83579/179505187-1b838d28-a152-4e98-891d-afecdf52655e.png">

What am I missing?

**Comment by kotp** [comment]
It is great that the tests are passing.  But I can not know what might be missing if I can not see the failing test(s), and probably the code.

One of the best ways to do this is by requesting mentoring.

If you were to submit the exercise, request mentoring, then hopefully the mentor can help determine what is going on with the solution in relation to the tests.

**Comment by belgoros** [comment]
@kotp thanks a lot for your feedback. I've just requested mentoring, the displayed error message does not provide enough information about the reason of fail:
<img width="466" alt="Screenshot 2022-07-19 at 08 05 01" src="https://user-images.githubusercontent.com/83579/179677524-62f82fc5-6d2d-41ce-b907-fe7a98797d1a.png">

**Comment by belgoros** [comment]
Yeah, just saw the same error saying about the timeout. After re-running, all is green. Thank you, will close the issue.
